docs/bbs/pam.md

```diff
@@ -30,3 +30,73 @@ Animations in Birth by Sleep do not need to be aligned in any way.
+
+The meanings of the set bits are as follows:
+
+| Bit Set | Description
+|---------|---------------
+| 0       | `translation.x` exists
+| 1       | `translation.y` exists`
+| 2       | `translation.z` exists
+| 3       | `rotation.x` exists
+| 4       | `rotation.y` exists
+| 5       | `rotation.z` exists
+| 6       | `scale.x` exists
+| 7       | `scale.y` exists
+| 8       | `scale.z` exists
+
+## Channel Data
+After the flags comes the actual animation data. It is ordered by bones, meaning first all channels for Bone0, then all channels for Bone1 and so on. The channels are also ordered as the table in channel flags section. If a flag for a channel was not set, then it is skipped and there is no data for it.
+
+The keyframe data (if exists) is quantized and stored as unsigned shorts. The maximum and minimum values are stored for each channel which are used for dequantization.
+
+### Channel Header
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 00 | float  | Maximum value for this channel|
+| 04 | float  | Minimum value for this channel|
+| 08 | byte   | Keyframe count (animation frame count <= 255) |
+| 08 | ushort | Keyframe count (animation frame count > 255) |
+
+## Keyframe Data
+After the channel header the keyframe data starts. The data storage also changes here depending on keyframe counts. So it will be divided into 3 categories for ease of understanding.
+
+### Keyframe Count = 0
+This channel uses a constant value throughout the animation. The maximum and minimum values in the channel header will be the same. Either one of these can be used as the constant value of this channel. There will be no keyframe data.
+
+### Keyframe Count = Animation Frame Count
+This channel has a key for every frame of this animation. So the frame ids of keyframes will not be stored. Frame id will start from 0 and will be incremented after each entry.
+
+#### Keyframe Entry
+
+| Variable Type | Description |
+|---------------|-------------|
+| ushort | Quantized value of this keyframe|
+
+### Keyframe Count < Animation Frame Count
+This channel has only keys for specific frames. Therefore the frame id is also stored before the value for each entry.
+
+#### Keyframe Entry
+
+| Variable Type | Description |
+|---------------|-------------|
+| byte   | Frame id of this keyframe (animation frame count <= 255)|
+| ushort | Frame id of this keyframe (animation frame count > 255)|
+| ushort | Quantized value of this keyframe|
+
+## Implementation and Other Uses
+An animation parser/viewer can be made using the documentation above. Although all the information was gathered from BBS, it seems to hold true for DDD. I have implemented this in my [tool for DDD](https://forum.xentax.com/viewtopic.php?p=169270&sid=51dbc614987b03be478bbcb5b90665a4#p169270) remaster on PS4.
```

Comment:
I would personally prefer to specify _"There is a tool for DDD"_ with the same link, so the documentation would be more generic.

Reply:
Alternately, "There is a tool created by Akderebur"  works, as it both gives you express credit, and also makes the docs more generic

Reply:
Good point @Tomk1004 . I agree that yours is a better resolution.

Reply:
I have made the changes and also fixed the error that thamstras pointed out.